backend/app/Actions/Visitors/GetVisitorsBounceRateByParameterAction.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\Visitors;
+
+use App\DataTransformer\Visitors\BounceRateVisitors;
+use App\Repositories\Contracts\TableVisitorsRepository;
+
+final class GetVisitorsBounceRateByParameterAction
+{
+    private $tableVisitorsRepository;
+
+    public function __construct(TableVisitorsRepository $tableVisitorsRepository)
+    {
+        $this->tableVisitorsRepository = $tableVisitorsRepository;
+    }
+
+    public function execute(GetVisitorsBounseRateByParameterRequest $request): GetVisitorsBounseRateByParameterResponce
+    {
+        $parameter = $request->parameter();
+        switch ($parameter) {
+            case 'city':
```

Comment:
Consider use const, in that way you will have all variants defined in one place. For example in this particular example, you have same string duplicate 3 times